tensorflow/compiler/xla/service/gpu/gpu_executable.cc

```diff
@@ -378,7 +378,7 @@ static Status CheckAlignment(const BufferAllocation& allocation,
   if (!buffer.is_null() &&
       reinterpret_cast<uintptr_t>(buffer.opaque()) % expected_alignment != 0) {
     return InternalError(
-        "Address of buffer %d must be a multiple of %x, but "
+        "Address of buffer %d must be a multiple of 0x%x, but "
```

Comment:
This is not a pointer.

Reply:
print with prefix "0x'" will more clearly to point it'is a hex number.